Added Theme Arc Dark

xypnox opened the pull request:
Hi this is arc dark theme. i hope to create and add more themes. Hope you like this one. I have also included a screenshot. It is based on the gnome theme Arc( [Arc](https://github.com/horst3180/Arc-theme) )

xypnox committed:
Added Theme Arc Dark

totoroot commented:
Hey there,

Seeing as PRs as far back as April 2017 were not addressed and the original maintainer does not seem to be active anymore, I decided to create a new repository and squash the commit history of all previous contributions to the original repository into one commit. I made sure to add a disclaimer that links to the original repository and include previous contributions.

If you wish your changes to be added to the [newly created repository](https://github.com/totoroot/ghostwriter-themes) please open up a PR and I will add you to the list of contributors if you wish so.

If @jggouvea or anyone else has a problem with me taking over maintaining these community themes or wishes parts of the newly created repository to be removed, please open an issue and tell me about it.

Cheers,
Matthias

xypnox commented:
Hi, I stopped using ghostwriter after a while. I have now switched to VSCode + [Dendron](https://www.dendron.so).

For me to open a PR in the new repo, I would have to fork it again and replicate the commits again.

As I have already switched the editors I do not have an incentive to do so. I apologize for the lack of effort.

If someone else wants to replicate the theme please feel free to duplicate the code to the new repo.
